packages/ui-components/src/components/form/Button.vue

```diff
@@ -8,31 +8,25 @@
     :class="buttonClasses"
     :disabled="isDisabled"
     role="button"
+    :style="
```

Comment:
There will be new shadows defined, but these have not been done yet. Inline until then.